package-lock.json

```
@@ -34,7 +34,7 @@
         "nock": "^13.5.6"
       },
       "engines": {
-        "node": ">=18"
+        "node": ">=22.0.0"
```

To mirror the `engines` suggestion in the `package.json`
```suggestion
        "node": ">=22 <23"
```

---

Discussed in private